### PR TITLE
ted talk with new CSS api added, 100% functional

### PR DESCRIPTION
**Pull request by gonzalezjc**
new CSS added, Ted-talk api component added and integrated, 100% responsive in the tests I run

### CODE REVIEW

**Review by YassinePerso — COMMENTED** [review]
Okay for me waiting for a second review

**Review by thgiolito — APPROVED** [review]

**Pull request merged**
